cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/LocalMetricsCollectionService.java

```diff
@@ -97,8 +99,11 @@ public void run() {
         // Only do cleanup if the underlying table doesn't supports TTL.
```

Comment:
it makes sense to add a comment instead of this one that we do cleanup only in LocalMetricsCollectionService - where we know ttl is not supported by underlying data store